components/common/modal/modalContent/ShareModalContent.tsx

```diff
@@ -0,0 +1,88 @@
+import * as S from "./ShareModalContent.styled";
+import { useEffect } from "react";
+import icon_share_kakao from "@/public/image/icon/icon_share_kakao.svg";
+import icon_share_facebook from "@/public/image/icon/icon_share_facebook.svg";
+import icon_share_link from "@/public/image/icon/icon_share_link.svg";
+import Image from "next/image";
+
+const SHARE_OPTION = {
+  keyId: [1, 2, 3],
+  img: [icon_share_kakao, icon_share_facebook, icon_share_link],
+  imgAlt: ["카카오톡 아이콘", "페이스북 아이콘", "링크복사 아이콘"],
+  buttonName: ["카카오톡", "페이스북", "링크 복사"],
+};
```

Comment:
데이터를 관리하기 위한 최적의 자료구조를 잘 선택하는 것도 중요합니다! 
저라면 아래처럼 만들어서 쓸 것 같아요

```suggestion
const SHARE_OPTIONS = [
  { keyId: 1, img: icon_share_kakao, imgAlt: "카카오톡 아이콘", buttonName: "카카오톡"},
  ...
];
```